src/widget/wbeatspinbox.cpp

```diff
@@ -297,16 +297,16 @@ bool WBeatSpinBox::event(QEvent* pEvent) {
 }
 
 void WBeatSpinBox::keyPressEvent(QKeyEvent* pEvent) {
-    // Return & Enter keys apply current value.
-    // Return, Enter and Escape send a Shift+Tab event in order to move focus
-    // to a library widget. In official skins this would be the tracks table.
+    // By default, Return & Enter keys apply current value.
+    // Here, Return, Enter and Escape apply and move focus to tracks table.
     if (pEvent->key() == Qt::Key_Return ||
             pEvent->key() == Qt::Key_Enter ||
             pEvent->key() == Qt::Key_Escape) {
         QDoubleSpinBox::keyPressEvent(pEvent);
-        QKeyEvent backwardFocusKeyEvent =
-                QKeyEvent{QEvent::KeyPress, Qt::Key_Backtab, Qt::NoModifier};
-        QApplication::sendEvent(this, &backwardFocusKeyEvent);
+        auto* libraryFocusCO = ControlObject::getControl("[Library]", "focused_widget");
```

Comment:
This can be done at once via:
ControlObject::set(ConfigKey(Library]", "focused_widget"), static_cast<double>(FocusWidget::TracksTable));
The cast should be double to avoid the implicit double conversion.
The same below.

Reply:
oh, thank you!
I wonder why, in other places, we first **get** the CO, then check for !nullptr before setting it, while this is already implemented in [ControlObject::set()](https://github.com/mixxxdj/mixxx/blob/12aec9b279a3ff398b1aa92b48e4635b0295d702/src/control/controlobject.cpp#L105-L110). Example